enterprise/app/controllers/api/v1/accounts/applied_slas_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Api::V1::Accounts::AppliedSlasController < Api::V1::Accounts::EnterpriseAccountsController
+  include Sift
+  include DateRangeHelper
+
+  before_action :set_sla_responses, only: [:metrics]
+  before_action :check_admin_authorization?
+
+  def metrics
+    @total_applied_slas = @sla_responses.count
+    @number_of_sla_breaches = @sla_responses.missed.count
```

Comment:
Calculate the hit percentage in the front end,

```
((total_applied_slas - number_of_sla_breaches) / total_applied_slas) * 100
```

Reply:
@pranavrajs @sojan-official Need your thoughts here?

Reply:
i think we can return the percentage as well along with the APIs . would be easier for the API consumers

Reply:
@sojan-official Done.